app/static/js/base.js

```diff
@@ -16,3 +16,44 @@ $(document).ready(function() {
         $("#userSelectForm").submit();
     });
 });
+
+function validatePhoneNumber(editButtonId, phoneInputId, username) {
+    var isvalid = null
+    if ($(editButtonId).html() === 'Edit') {
+        $(editButtonId).html("Save");
+        $(phoneInputId).focus();
+    } else {
+        // Save the phone number
+        var phoneInput = $(phoneInputId);
+        isvalid = phoneInput.val().replace(/\D/g,"").length === 10;
+        let isempty = phoneInput.val().replace(/\D/g,"").length === 0;
+        if (!(isvalid || isempty)) {
+            phoneInput.addClass("invalid");
+            window.setTimeout(() => phoneInput.removeClass("invalid"), 1000);
+            phoneInput.focus()
+            isvalid = false;
+            return isvalid;
+        }
+        if (isvalid == true){
```

Comment:
We want to be able to save an empty phone number, so we can just get rid of this isvalid check.